XCUITests/BookmarkingTests.swift

```diff
@@ -19,26 +19,29 @@ class BookmarkingTests: BaseTestCase {
     }
     
     private func bookmark() {
-        navigator.goto(BrowserTabMenu)
-        app.collectionViews.cells["AddBookmarkMenuItem"].tap()
+        app.buttons["TabLocationView.pageOptionsButton"].tap()
```

Comment:
did this work? if so we should close the bug

Reply:
Actually, I think it would be better to create a scene in Fxscreengraph for this menu list, then we can do things like navigator.goto(sharemenu) to open the menu list. let's discuss this tomorrow.